### PR TITLE
created section

### PR DESCRIPTION
**Pull request by ElShaer398**
Portfolio-setup-and-mobile-first project at Microverse.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @ElShaer398,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- Kindly fix your Pull Request title as requested [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md), it should have a descriptive title telling you what changes are inside. Also please fix the Pull Request description section, it should have a short summary describing the introduced changes. You can change it with the menu  I marked below on the image.
Screenshot 2023-02-25 at 1 02 33 AM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by ouasamine — COMMENTED** [review]
Hi @ElShaer398 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where the README file is not properly customised.

**Friendly reminder**: You should make sure that your repository looks professional before review. Please take a step back and apply the 5 simple rules from this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). 
**Please make sure to customise the README file and add valid data related to the current project in all sections, and make sure to delete the "live demo" section as you didn't deploy your project yet**.

Example: instead of having placeholder text for the "future features" add actual features that you are intending to add in the future and remove the description lines as they are there to guide you and are not intended for users of your repository, as described below 🔽 

Change this
>  [new_feature_1]
  [new_feature_2]
  [new_feature_3]

To this for example
> Create a desktop version
  Deploy the project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -139,5 +139,17 @@ <h3 class="connect" id="connect">LET'S CONNECT</h3>
...
+    <h2 class="contact-form-title">Contact me</h2>
+    <p class="contact-form-content">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+    <form  class="form" action="https://formspree.io/f/xqkjrbdz" method="post">
+      <input type="text" id="name" class="form-items" name="user_name" maxlength="30" value="Yeremius" placeholder="name" required/>
+      <input type="email" id="email" class="form-items" name="user_email" value="natajayanj14@gmail.com" placeholder="email" required/>
+      <textarea id="textarea" class="form-items" name="textarea" placeholder="Write your message here" maxlength="500" required></textarea>
+      <input type="submit" id="submit" class="form-items" value="Get In Touch">
+    </form>
+    <img class="home-indicator" src="images/home-indicator.png" alt="home indicator">
+  </section>
```
Comment (didierganthier) [inline index.html:153]: [OPTIONAL] Hey 👋🏽 avoid putting values in the inputs, it's better to have placeholders and let the user add the values himself. 😉

**Review by didierganthier — APPROVED** [review]
Hi @ElShaer398 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**